container-terminal.js

```diff
@@ -82,9 +82,12 @@
                         rows: '=',
                         cols: '=',
                         screenKeys: '=',
-                        autofocus: '=?'
+                        autofocus: '=?',
+                        status: '='
```

Comment:
would be good to have a comment here explaining that it is not an input and that it is for reporting status back to the parent scope.